Dockerfile.assisted-service-build

```diff
@@ -1,31 +1,31 @@
-FROM quay.io/centos7/postgresql-12-centos7:latest
-
-COPY --from=quay.io/goswagger/swagger:v0.28.0 /usr/bin/swagger /usr/bin/goswagger
-COPY --from=quay.io/edge-infrastructure/swagger-codegen-cli:2.4.18 /opt/swagger-codegen-cli /opt/swagger-codegen-cli
-COPY --from=quay.io/app-sre/golangci-lint:v1.37.1 /usr/bin/golangci-lint /usr/bin/golangci-lint
+FROM quay.io/centos/centos:stream9
 
 ENV GOPATH=/go
 ENV GOROOT=/usr/local/go
-ENV PATH=$PATH:$GOROOT/bin:$GOPATH/bin
-
-USER 0
-
-RUN mkdir build && chmod g+xw -R build/
-
-RUN yum install -y --setopt=skip_missing_names_on_install=False \
-        gcc genisoimage git libvirt-client libvirt-devel java && yum clean all
+ENV VIRTUAL_ENV=/opt/venv
+# A directory in the path with write permission even for non-root users
+ENV TOOLS=/tools/
+ENV PATH="$VIRTUAL_ENV/bin:$GOROOT/bin:$GOPATH/bin:$TOOLS:$PATH"
+ENV CGO_CFLAGS="-g -O2 -Wno-return-local-addr"
 
+COPY --from=quay.io/app-sre/golangci-lint:v1.37.1 /usr/bin/golangci-lint /usr/bin/golangci-lint
+COPY --from=quay.io/goswagger/swagger:v0.28.0 /usr/bin/swagger /usr/bin/goswagger
+COPY --from=quay.io/edge-infrastructure/swagger-codegen-cli:2.4.18 /opt/swagger-codegen-cli /opt/swagger-codegen-cli
 COPY --from=registry.ci.openshift.org/openshift/release:golang-1.17 /usr/bin/gotestsum /usr/bin/make /usr/bin/
 COPY --from=registry.ci.openshift.org/openshift/release:golang-1.17 /usr/local/go /usr/local/go
+COPY --from=quay.io/openshift/origin-cli:latest /usr/bin/oc /usr/bin
+COPY --from=quay.io/operator-framework/upstream-opm-builder:v1.16.1 /bin/opm /bin
```

Comment:
also this one should be substituted after this gets in

Reply:
The build image shouldn't be part of the CI. It's a huge image.